### ISSUE TITLE
Format width is not obeyed when using ColoredLevelConfig

### ISSUE THREAD

**Issue by Mishio595** [daboross/fern#34]
This is my current configuration for fern, `file_out` logs with the correct `level_width` of 8, but `term_out` completely ignores the width.

I have also tested to see if it follows alignment formatting marks, and it does not.
```rust
    let colors = ColoredLevelConfig::new()
        .trace(Color::Magenta)
        .debug(Color::Cyan)
        .info(Color::Green)
        .warn(Color::Yellow)
        .error(Color::Red);

    let term_out = fern::Dispatch::new()
        .format(move |out, message, record| {
            out.finish(format_args!(
                "{time}  {level:level_width$}{target:target_width$}\t> {msg}",
                time = chrono::Local::now().format("%Y-%m-%d %H:%M:%S"),
                level = colors.color(record.level()),
                target = record.target(),
                msg = message,
                level_width = 8,
                target_width = 60
            ))
        })
        .chain(std::io::stdout())
        .into_shared();

    let file_out = fern::Dispatch::new()
        .format(move |out, message, record| {
            out.finish(format_args!(
                "{time}  {level:level_width$}{target:target_width$}\t> {msg}",
                time = chrono::Local::now().format("%Y-%m-%d %H:%M:%S"),
                level = record.level(),
                target = record.target(),
                msg = message,
                level_width = 8,
                target_width = 60
            ))
        })
        .chain(fern::log_file("output.log").expect("Unable to access log file"))
        .into_shared();

    fern::Dispatch::new()
        .level(log::LevelFilter::Info)
        .level_for("momiji", log::LevelFilter::Debug)
        .chain(term_out)
        .chain(file_out)
        .apply().expect("Failed to apply fern settings");
```

**Comment by earwig** [comment]
I think this is fixed by #32, but it hasn't been released yet.

Hey @daboross, would it be possible to release a version 0.5.7 with this fix? Thank you!

**Comment by daboross** [comment]
Can do - thanks for the reminder!

**Comment by aldanor** [comment]
This can be closed since 0.5.7 is out?

**Comment by daboross** [comment]
Thanks for finding that. Closing this!
